FitGoal/FitGoal/UserProfileConfiguration/ViewControllers/WalkthroughPageViewController.swift

```diff
@@ -142,6 +148,15 @@ class WalkthroughPageViewController: UIPageViewController, UIPageViewControllerD
             currentIndex += 1
             pageControl.currentPage = currentIndex
             self.setViewControllers([walkthroughViewControllers[currentIndex]], direction: .forward, animated: true, completion: nil)
+            changeTitle(forLabel: viewTitleLabel, vcIndex: currentIndex)
+        }
+    }
+    
+    private func changeTitle(forLabel label: UILabel, vcIndex index: Int) {
```

Comment:
The name of the second parameter should be clearer, something like `viewControllerIndex`. Not need to change it now but keep it in mind.